index.html

```diff
@@ -46,5 +46,29 @@ <h2>Question ?</h2>
         </ul>
       </section>
     </main>
+    <footer>
+      <nav>
+        <ul class="container-nav">
+          <li>
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/74/74807.png"
+              alt=""
```

Comment:
You could add an alt-text, wich is important for the accessibility and if the icon can't be shown, this text will be provided.